portal/migrations/versions/426c876ed746_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: 426c876ed746
+Revises: b92dc277c384
+Create Date: 2017-05-12 11:57:29.387852
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '426c876ed746'
+down_revision = 'b92dc277c384'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('organizations', sa.Column('default_locale_id', sa.Integer(), nullable=True))
```

Comment:
Curious about the name "default_locale_id".  Not sure it has to change, but why not just call it "locale_id"?  If the value gets changed, the column name will be misleading, no?